### DISCUSSION TITLE
Arbitrary values for col-span and row-span not working?

### DISCUSSION THREAD

**Discussion by MartinHughes-BPC** [tailwindlabs/tailwindcss#6460] [Help] [answered]
Could somebody check this out for me in tw 3.01 - I can't get arbitrary values to work for col-span or row-span.

Here's a quick play: https://play.tailwindcss.com/8B3t6kqErv

Thanks,
Martin.

**Comment by adamwathan** [comment, marked as answer]
Hey! Here's a working version:

https://play.tailwindcss.com/2KY2C1YOJH

This is a mistake in the docs, the prefix for these classes is just "row" because they just set the `grid-row` property. Will get it updated!

**Reply by MartinHughes-BPC** [reply]
Magic! Thanks Adam 👍
